docs/antrea-proxy.md

```diff
@@ -42,14 +42,20 @@ the introduction of `proxyAll`, Antrea relied on userspace kube-proxy, which is
 no longer actively maintained by the K8s community and is slower than other
```

Comment:
I feel we should change all "AntreaProxy" to "Antrea Proxy", except when it refers to the "AntreaProxy" feature gate, or any other code name.

Reply:
How about updating this in another PR?

Reply:
Sure.